mtags/src/main/scala-2/scala/meta/internal/pc/HoverProvider.scala

```diff
@@ -39,6 +43,28 @@ class HoverProvider(val compiler: MetalsGlobal, params: OffsetParams) {
         val tree = typedHoverTreeAt(pos, unit)
         (pos, tree)
     }
+
+    def report =
+      Report(
+        "empty-hover",
+        s"""|${params.uri().toString()}
+            |pos: ${pos.toLsp}
+            |
+            |is error: ${if (tree.tpe != null) tree.tpe.isErroneous
```

Comment:
Could you move the if else code block to a def above? Might be a bit easier to read and understand